code/game/gamemodes/blob/powers.dm

```diff
@@ -95,12 +95,13 @@
 	playsound(B.loc, 'sound/effects/splat.ogg', 50, 1)
 	blobber.overmind = src
 	blobber.update_icons()
-	blobber.AIStatus = AI_OFF
+	blobber.notransform = 1 //stop the naut from moving around
```

Comment:
After  some discussion in coderbus, this is the right var for what changling needs it for.